docs/tasks.rst

```diff
@@ -3907,6 +3932,16 @@ Options
 
 	 Set to Serial to force serial mode on all jobs. Parallel is the default.
 
+``-o inject_namespaces INJECTNAMESPACES``
+	 *Optional*
+
+	 If True, the package namespace prefix will be automatically added to objects and fields for which it is present in the org. Defaults to True.
+
+``-o drop_missing_schema DROPMISSINGSCHEMA``
+	 *Optional*
+
+	 Set to True to skip any missing objects or fields instead of stopping with an error.
+
 **load_custom_settings**
 ==========================================
 
```

Comment:
Need to update this with the new generate_dataset_mapping option.

Reply:
Done!